lib/oci_utils/kvm/virt.py

```diff
@@ -822,6 +841,9 @@ def create_virtual_network(**kargs):
             An instance of StringIO populated with command lines which can be used to persist the network creation, None on failure
     """
 
+    _instance_shape = InstanceMetadata()['instance']['shape']
+    _is_bm_shape = _instance_shape.startswith('BM')
+
     _persistence_script = StringIO.StringIO()
     _persistence_script.write('#/bin/bash\n')
```

Comment:
I know this will change since we will start using systemd, but this should #!/bin/bash

Reply:
fixed